tests/rptest/tests/cloud_retention_test.py

```diff
@@ -165,7 +165,7 @@ def test_gc_entire_manifest(self):
         further uploads from taking place.
         """
         small_segment_size = 1024 * 1024
-        num_partitions = 5
+        num_partitions = 16
```

Comment:
nit: Since we increased the number of partitions should we also write more data? I don't think it matters though, as we'll still GC everything in the cloud.

Reply:
no, you're right, I should have updated this.  Will do.